playground/nuxt.config.ts

```diff
@@ -2,7 +2,7 @@ export default defineNuxtConfig({
   extends: ['./theme'],
   modules: [
     '@nuxt/content',
-    '../src/module',
+    '@nuxtjs/tailwindcss',
```

Comment:
Advocating for this as we have the module imported as function in the layer, but this would allow the module build/stub to be tested

Reply:
(will get back to this later when we sort `pnpm-workspace.yaml`